packages/clay-shared/src/Portal.tsx

```diff
@@ -41,15 +40,17 @@ export const ClayPortal: React.FunctionComponent<
 				? containerRef.current
 				: closestParent;
 
-		if (elToMountTo && portalRef.current) {
-			if (otherProps?.className) {
-				portalRef.current.classList.add(otherProps.className);
-			}
-			if (otherProps?.id) {
-				portalRef.current.id = otherProps.id;
-			}
+		if (elToMountTo) {
+			const myDiv = document.createElement('div');
+
+			ReactDOM.render(<div {...otherProps}></div> , myDiv, () => {
```

Comment:
`ReactDOM.render` does not allow you to render directly in the `document.body` element. Thus, I had to create a temporal div `myDiv`, and assign its firstChild to the `portalRef.current`. This should result in the same that we had before while doing `elToMountTo.appendChild(portalRef.current);`.